packages/scaffold-config/src/frameworks.ts

```diff
@@ -38,7 +42,7 @@ export async function isDependencyInstalled (dependency: Cypress.CypressComponen
     return {
       dependency,
       detectedVersion: pkg.version,
-      loc,
+      loc: path.parse(packageFilePath).dir,
```

Comment:
I am curious if this works for namespaced packages like `@vitejs/plugin-react` which also lacks `package.json` in `exports`: https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/package.json

Can you verify that works as expected? Not sure how `dir` will resolve.

Also - do we even use `.loc`? If not, we could just remove it entirely.

Reply:
Yeah I don't see this being used anywhere anyway. I'll remove it